src/commands/synthetics/upload-application-command.ts

```diff
@@ -106,7 +107,11 @@ export class UploadApplicationCommand extends Command {
       removeUndefinedValues({
         apiKey: process.env.DATADOG_API_KEY,
         appKey: process.env.DATADOG_APP_KEY,
+        configPath: process.env.DATADOG_SYNTHETICS_CONFIG_PATH,
```

Comment:
I believe you'll need to do a similar trick above -- we want to take the `process.env.DATADOG_SYNTHETICS_CONFIG_PATH` env var into consideration when calling `resolveConfigFromFile`.

Reply:
oh right, forgot about this one !